gprofiler/metadata/application_metadata.py

```diff
@@ -87,4 +88,19 @@ def make_application_metadata(self, process: Process) -> Dict[str, Any]:
             execfn = f"error: {e.__class__.__name__}"
         md["execfn"] = execfn
 
+        try:
```

Comment:
Consider refactoring the try-except to a function instead of duping it for every call to granulate_utils

Reply:
Hai